data/reporter.js

```diff
@@ -106,15 +108,10 @@ var initialize = function(data) {
 };
 
 self.port.on("app_e10s_enabled", function(enabled) {
-    $("#appE10sEnabled").empty();
-    var e10s_link = document.createElement("a");
-    e10s_link.setAttribute("href", "#");
-    e10s_link.addEventListener("click", function() {
+    $(".app-e10s-enabled").html('<a href="#">Multi-process</a> is ' + (enabled ? '' : 'not ') + 'enabled.');
+    $(".app-e10s-enabled").on("click", function() {
```

Comment:
Nice. It's not as explicit for AMO reviewers though, I plead guilty writing that code. :)